kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml

```diff
@@ -346,12 +350,13 @@
             </div>
             <div xml:id="kzz7yh-f24075-Dd1e618">
                <head xml:id="kzz7yh-f24075-Hd1e620">Quaestio 4</head>
+               <head type="question-title">Utrum corpora quae demones assumunt sint perfectibilia ab animal intellectiva<!--endings--></head>
                <p xml:id="kzz7yh-f24075-d1e622">
       ¶ Quaestio IIII. 
-      <lb ed="#V" n="42"/>QUarto queritur. vtrum 
-      copo<lb ed="#V" n="43" break="no"/>ra quae 
+      <lb ed="#V" n="42"/>QUarto queritur vtrum 
+      corpo<lb ed="#V" n="43" break="no"/>ra quae 
       demo<lb ed="#V" n="44" break="no"/>nes assumunt sint perfectibilia ab anima 
-      intel<lb ed="#V" n="45" break="no"/>llectiua. Et videtur quod sic. Quia vt habitum est 
+      intel<lb ed="#V" n="45" break="no"/>llectiua<!--ending?-->. Et videtur quod sic. Quia vt habitum est 
```

Comment:
```suggestion
      intel<lb ed="#V" n="45" break="no"/>llectiua. Et videtur quod sic. Quia vt habitum est 
```

Reply:
It's 1st dec. ablative, singular, feminine matching with "anima" after the preposition "ab."